src/shared_utilities/Domains.jl

```diff
@@ -1209,6 +1209,9 @@ end
 function landsea_mask(domain::Domains.AbstractDomain; kwargs...)
```

Comment:
this function for SphericalShell, SphericalSurface, HybridBox, Plane

new method for Point, Column that returns nothing